### DISCUSSION TITLE
标题栏按钮功能，歌单右键功能

### DISCUSSION THREAD

**Discussion by pgxppp** [linuxdeepin/developer-center#3092] [Features Request & Ideas | 特性请求 & 头脑风暴]
标题栏按钮功能增加“添加音乐路径”，歌单右键功能增加“添加音乐”
git clone -b pgxppp https://gitee.com/pgxppp/deepin-music.git
另：希望提供丰富歌词解析功能并提供桌面动感歌词功能

**Comment by deepin-CD** [comment]
We will seriously consider your suggestion. Thank you for your suggestion.

**Comment by meiyixiang** [comment]
你好这些功能我们在新的音乐中已经支持，你可以升级到最新的版本。
